src/core/ContentSnapshot.js

```diff
@@ -23,18 +23,19 @@
         game.stage.addChildAt(this._graphicFill, 0);
 
         // Create the game texture
-        this._texture = new Phaser.RenderTexture(game, game.width, game.height);
+        this._texture = game.add.renderTexture(game.width, game.height);
```

Comment:
I would keep the constructor, just so the instance won't be added to the `game.world` and then returned.